benchmark/benchmark-tidb-using-ch.md

```diff
@@ -0,0 +1,171 @@
+---
+title: How to Run CH Benchmark Test on TiDB
+summary: Learn how to run CH Benchmark test on TiDB.
+---
+
+# How to Run CH Benchmark Test on TiDB
+
+This document describes how to test TiDB using CH Benchmark.
+
+CH Benchmark is a mixed workload containing both [TPC-C](http://www.tpc.org/tpcc/) and [TPC-H](http://www.tpc.org/tpch/) tests, and it is the most common workload to test HTAP systems. For more information, see [The mixed workload CH-benCHmark](https://research.tableau.com/sites/default/files/a8-cole.pdf).
+
+Before running the CH Benchmark test, you need to deploy [TiFlash](/tiflash/tiflash-overview.md) first, which is TiDB's HTAP component. After you deploy TiFlash and [create TiFlash replicas](#create-tiflash-replicas), TiKV will replicate the latest data of TPC-C online transactions to TiFlash in real time, and the TiDB optimizer will automatically push down OLAP queries from TPC-H workload to the MPP engine of TiFlash for efficient execution.
+
+The CH Benchmark test in this document is implemented based on [go-tpc](https://github.com/pingcap/go-tpc). You can download the test program using the following [TiUP](/tiup/tiup-overview.md) command:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup install bench
+```
+
+For detailed usage of the TiUP Bench component, see [TiUP Bench](/tiup/tiup-bench.md).
+
+## Load data
+
+### Load TPC-C data
+
+**Loading data is usually the most time-consuming and problematic stage of the entire TPC-C test.**
+
+Taking 1000 warehouses as an example, you can execute the following TiUP command in shell for data load and test:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup bench tpcc -H 172.16.5.140 -P 4000 -D tpcc --warehouses 1000 prepare -T 32
```

Comment:
-H 172.16.5.140 Should we use a variable placeholder instead of 172.16.5.140？

Reply:
Added a sentence: "Note that you need to replace 172.16.5.140 and 4000 in this document with your TiDB host and port values."